### PR TITLE
chore: add bug report issue template

### PR DESCRIPTION
**Pull request by MaikoTan**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        - Koishi (independent)
+        - Others
+
+  - type: dropdown
+    id: os
+    attributes:
+      label: What operating system?
+      multiple: true
+      options:
+        - Windows 11 / 10 / 8
```
Comment (shigma) [inline .github/ISSUE_TEMPLATE/bug_report.yml:67]: Would it be better to separate different versions of Windows?

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+        - Other
+
+  - type: textarea
+    id: versions
+    attributes:
+      title: Versions
+      description: 填写你的环境信息
+      value: |
+        plugin: <!-- 出问题的插件版本 -->
+        koishi:
```
Comment (shigma) [inline .github/ISSUE_TEMPLATE/bug_report.yml:83]:
```suggestion
        Koishi:
```

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+  - type: dropdown
+    id: satori-agent
+    attributes:
+      label: What agent for Koishi / Satori?
+      multiple: true
+      options:
+        - Koishi Desktop
+        - Koishi Android
+        - Koishi (independent)
```
Comment (shigma) [inline .github/ISSUE_TEMPLATE/bug_report.yml:58]: independent -> standalone?

**Review by shigma — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 1 above)
Comment (MaikoTan) [reply .github/ISSUE_TEMPLATE/bug_report.yml:67]: Eeeeh, I remembered that we have told about it before?
You could only use NodeJS 12 on Windows 7 or something, no?

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 3 above)
Comment (MaikoTan) [reply .github/ISSUE_TEMPLATE/bug_report.yml:58]: Ah, yes.
Forgot this word.

Comment (MaikoTan) [inline .github/ISSUE_TEMPLATE/bug_report.yml:58]:
```suggestion
        - Koishi (standalone)
```

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 1 above)
Comment (shigma) [reply .github/ISSUE_TEMPLATE/bug_report.yml:67]: I mean distinguish between windows 8 / 10 / 11. Issues with windows 7 and nodejs 12 should still be possible in the future.

Comment (MaikoTan) [reply .github/ISSUE_TEMPLATE/bug_report.yml:67]: It looks like that you could install node 14 on win 8, and the latest version on 10 / 11?
If you distinct with this, I would do the change.

**Review by shigma — APPROVED** [review]

**Pull request merged**